Mathlib/Algebra/Order/Ring/Unbundled/Rat.lean

```diff
@@ -142,4 +142,13 @@ theorem abs_def (q : ℚ) : |q| = q.num.natAbs /. q.den := by
       mul_one, zero_mul] at hq
     rw [Int.natAbs_of_nonneg hq, num_divInt_den]
 
+theorem abs_def' (q : ℚ) : |q| = ⟨q.num.natAbs, q.den, q.den_ne_zero, q.reduced⟩ := by
+  rw [abs_def, ← Rat.mk_eq_divInt q.num.natAbs q.den q.den_ne_zero q.reduced]
+
+theorem abs_num_eq_num_natAbs (q : ℚ) : |q|.num = q.num.natAbs := by
```

Comment:
If I read correctly, there is an implicit coercion from `Nat` to `Int` on the right side. Wouldn't it be more natural to state
```
theorem abs_num (q : ℚ) : |q|.num =|q.num| := by 
```
?

(same question applies to `abs_def'`. I doubt that `natAbs` is useful as a public API)